eva-pipeline/src/test/java/embl/ebi/variation/eva/pipeline/jobs/VariantConfigurationTest.java

```diff
@@ -286,6 +303,180 @@ public void validLoadStats() throws JobExecutionException, IOException, IllegalA
+        String compressExtension = ".gz";
+        String outputDir = "/tmp";
+        File vepInput = new File(Paths.get(outputDir).resolve(VariantStorageManager.buildFilename(source)) + ".variants.preannot.gz");
+        File vepOutput = new File(Paths.get(outputDir).resolve(VariantStorageManager.buildFilename(source)) + ".variants.annot.gz");
+        String mockVep = VariantConfigurationTest.class.getResource("/mockvep.pl").getFile();
+
+        JobParameters parameters = new JobParametersBuilder()
+                .addString("input", input)
+                .addString("outputDir", outputDir)
+                .addString("dbName", dbName)
+                .addString("compressExtension", compressExtension)
+                .addString("compressGenotypes", "true")
+                .addString("includeSrc", "FIRST_8_COLUMNS")
+                .addString("aggregated", "NONE")
+                .addString("studyType", "COLLECTION")
+                .addString("studyName", source.getStudyName())
+                .addString("studyId", source.getStudyId())
+                .addString("fileId", source.getFileId())
+                .addString("opencga.app.home", opencgaHome)
+                .addString(VariantsStatsCreate.SKIP_STATS_CREATE, "true")
+                .addString(VariantsStatsLoad.SKIP_STATS_LOAD, "true")
+                .addString(VariantsAnnotLoad.SKIP_ANNOT_LOAD, "true")
+                .addString("vepInput", vepInput.toString())
+                .addString("vepPath", mockVep)
+                .addString("vepCacheVersion", "79")
+                .addString("vepCacheDirectory", "/path/to/cache")
+                .addString("vepSpecies", "homo_sapiens")
+                .addString("vepFasta", "/path/to/file.fa")
+                .addString("vepNumForks", "4")
+                .addString("vepOutput", vepOutput.toString())
+                .toJobParameters();
+
+        vepInput.delete();
+        vepOutput.delete();
+        assertFalse(vepInput.exists());  // ensure the pre annot file doesn't exist from previous executions
+        assertFalse(vepOutput.exists());  // ensure the annot file doesn't exist from previous executions
+        JobExecution execution = jobLauncher.run(job, parameters);
+        assertEquals(ExitStatus.COMPLETED.getExitCode(), execution.getExitStatus().getExitCode());
+        assertTrue(vepInput.exists());
+        assertTrue(vepOutput.exists());
+
+        BufferedReader outputReader = new BufferedReader(new InputStreamReader(new GZIPInputStream(new FileInputStream(vepOutput))));
+        
+        // Check output file contents
+        int numLinesRead;
+        String outputLine = outputReader.readLine();
+        for (numLinesRead = 0; outputLine != null; numLinesRead++) {
+            assertEquals(numLinesRead + " annotated", outputLine);
+            outputLine = outputReader.readLine();
+        }
+        
+        // Check output file length
+        assertEquals(300, numLinesRead);
```

Comment:
You're right, that was removed on the first tries because it wasn't compatible with how the code worked at that point, but I have checked it with the current code and it works fine. I'll restore it.